tools/stern/src/Stern/API.hs

```diff
@@ -228,6 +227,22 @@ sitemap = do
       Doc.optional
     Doc.response 200 "Account deleted" Doc.end
     Doc.response 400 "Bad request" (Doc.model Doc.errorModel)
+  delete "/teams/:tid" (continue deleteTeam) $
+    capture "tid"
+      .&. query "email"
+  document "DELETE" "deleteTeam" $ do
+    summary "Delete a team (irrevocable!) You can only delete teams with 1 user!"
+    Doc.notes "Email the userId's (to prevent copy/paste mistakes)"
```

Comment:
Not quite sure what this means. It seems like you need to provide the email address of the only team member, but what does it have to do with a "userId"?

Reply:
```suggestion
    Doc.notes "Email of the user (to prevent copy/paste mistakes)"
```

Reply:
> Not quite sure what this means. It seems like you need to provide the email address of the only team member, but what does it have to do with a "userId"?

it was meant to be user, thanks

Reply:
Then I'd either put it into the "email" parameter's description or turn it into a full sentence like "The email address of the user must be provided to prevent copy/paste mistakes".

Reply:
I just pushed that to the branch, hope that was okay. Once CI passed, we can merge (in my opinion).